deep-security-agent/tasks/install-linux.yml

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Install the Deep Security Agent (For Linux only)
+# *********************************************************************
+---
+# *********************************************************************
+# * Detect the platform architecture and package manager
+# *********************************************************************
+- name: Setting facts (apt) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.deb
+  when: ansible_pkg_mgr == "apt"
+
+- name: Setting facts (yum, zypper) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.rpm
+  when: >
+        ansible_pkg_mgr == "yum" or
+        ansible_pkg_mgr == "zypper"
+
+- name: Setting facts (svr4pkg) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.pkg.gz
+  when: ansible_pkg_mgr == "svr4pkg"
+
+- name: Setting facts (pkg5) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.p5p.gz
+  when: ansible_pkg_mgr == "pkg5"
+
+- name: Setting facts (Amazon) 
+  set_fact:
+    agent_relative_path: software/agent/amzn1/{{ ansible_architecture }}
```

Comment:
We may need to consider amzn2 as it will be official released soon.

Reply:
Add condition for Amazon Linux 2.